pkg/constants/constants.go

```diff
@@ -512,6 +512,12 @@ const (
 
 	// TrustedCABundleFile is the name of the file (and corresponding ConfigMap key) containing the merged CA bundle.
 	TrustedCABundleFile = "ca-bundle.crt"
+
+	// OverrideMachinePoolPlatformAnnottaion can be set to "true" on a MachinePool to bypass the validating admission
```

Comment:
```suggestion
	// OverrideMachinePoolPlatformAnnotation can be set to "true" on a MachinePool to bypass the validating admission
```

Reply:
Wow, I even fixed this typo in the actual variable name, but I guess either I didn't use F2 or F2 didn't sweep up the comment (sometimes it does, sometimes it doesn't). Thanks!